### ISSUE TITLE
[Maverick] Podcast screen "Only Show Favorites" toggle label missing

### ISSUE THREAD

**Issue by rblakesley** [infinitered/ignite#2197]
### Describe the bug

The "Only Show Favorites" toggle label at the top of the Podcast screen no longer appears on Mavericks beta 7.  It looks like the enclosing `<View style={[$rowLayout, $toggle]}>` is preventing the text from showing.

Android:
![Screenshot_20220922-121322](https://user-images.githubusercontent.com/17591275/191738550-3d3c7185-d3ef-47fd-9efb-3f873bd83176.png)

iOS:
![IMG_5143](https://user-images.githubusercontent.com/17591275/191739097-486cd58f-e94d-42a4-8ea2-0376b1cb1d27.PNG)

### Ignite version

Mavericks beta 7

### Additional info

```
rb$ npx ignite-cli doctor
System
  platform           darwin                                           
  arch               arm64                                            
  cpu                10 cores         Apple M1 Max                    
  directory          IgniteMaverick   /Users/rb/GitHub/IgniteMaverick 

JavaScript (and globally-installed packages)
  node               18.4.0       /Users/rb/.nvm/versions/node/v18.4.0/bin/node 
  npm                8.12.1       /opt/homebrew/bin/npm                         
    expo-cli         5.4.12                                                     
    npm              8.12.1                                                     
  yarn               1.22.19      /opt/homebrew/bin/yarn                        
    react-devtools   4.24.7                                                     
  pnpm               -            not installed                                 
  expo               46.0.10      managed                                       

Ignite
  ignite-cli         7.15.0       /Users/rb/.npm/_npx/e31027f3785124a8/node_modules/.bin/ignite      
  ignite src         build        /Users/rb/.npm/_npx/e31027f3785124a8/node_modules/ignite-cli/build 

Android
  java               11.0.16      /Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin/java 
  android home       -            /Users/rb/Library/Android/sdk                                        

iOS
  xcode              13.4.1     
  cocoapods          1.11.3       /opt/homebrew/bin/pod 
```

**Comment by frankcalise** [comment]
@rblakesley thanks for the report! We'll get that fixed up.

**Comment by yulolimum** [comment]
Fixed with #2202
